tests/value/shapley/test_montecarlo.py

```diff
@@ -28,36 +26,41 @@
 
 # noinspection PyTestParametrized
 @pytest.mark.parametrize(
-    "num_samples, fun, rtol, n_iterations, kwargs",
+    "num_samples, fun, rtol, kwargs",
     [
-        (12, permutation_montecarlo_shapley, 0.1, 10, {}),
+        (12, ShapleyMode.PermutationMontecarlo, 0.1, {"done": MaxUpdates(10)}),
         # FIXME! it should be enough with 2**(len(data)-1) samples
-        (8, combinatorial_montecarlo_shapley, 0.2, 2**10, {}),
-        (12, truncated_montecarlo_shapley, 0.1, 10, {"coordinator_update_period": 1}),
-        (12, owen_sampling_shapley, 0.1, 4, {"max_q": 200, "method": "antithetic"}),
-        (12, owen_sampling_shapley, 0.1, 4, {"max_q": 200, "method": "standard"}),
+        (8, ShapleyMode.CombinatorialMontecarlo, 0.2, {"done": MaxUpdates(2**10)}),
+        (
+            12,
+            ShapleyMode.TruncatedMontecarlo,
+            0.1,
+            dict(
+                coordinator_update_period=1,
+                worker_update_period=0.5,
+                done=MaxUpdates(500),
+                truncation=NoTruncation(),
```

Comment:
Testing TMCS with truncations is tricky. Lower values lose a lot of accuracy, even if rank is preserved. I think we might need more tests for that. I still need to add tests for the truncation methods themselves 😓 . I have done some eyeball-testing with pdb in `ray debug`, but suppose that's not enough 😄 .